pincer/objects/guild/member.py

```diff
@@ -36,10 +36,10 @@ class BaseMember(APIObject):
     hoisted_role: APINullable[:class:`~pincer.utils.snowflake.Snowflake`]
         The user's top role in the guild.
     """
-    deaf: bool
     joined_at: Timestamp
-    mute: bool
     roles: List[Snowflake]
+    deaf: bool = MISSING
+    mute: bool = MISSING
```

Comment:
Why? :thinking:

Reply:
The user command interaction doesn't return deaf/mute for some reason

Reply:
Interesting